sql/99-patches.sql

```diff
@@ -241,5 +241,58 @@ $$
 $P$
 );
 
+--------------------------------------------------------------------------------
+-- LOL 3.22b: Add crs_ttl_inst_protect table
+--
+-- Added in LOL-3.22b
+--------------------------------------------------------------------------------
+
+PERFORM _patches.apply_patch(
+    'LOL 3.22p: Add crs_ttl_inst_protect table',
+    $P$
+DO $$
+BEGIN
+
+  CREATE TABLE bde.crs_ttl_inst_protect (
+      id INTEGER PRIMARY KEY,
+      tin_id INTEGER NOT NULL,
+      protect_start DATE,
+      protect_end DATE,
+      protect_reference VARCHAR(100)
+  );
+
+  ALTER TABLE bde.crs_ttl_inst_protect OWNER TO bde_dba;
+
+  REVOKE ALL
+      ON TABLE bde.crs_ttl_inst_protect
+      FROM public;
+
+  GRANT UPDATE, INSERT, DELETE
+      ON TABLE bde.crs_ttl_inst_protect
+      TO bde_admin;
+
+  GRANT SELECT
+      ON TABLE bde.crs_ttl_inst_protect
+      TO bde_user;
+
+  -- If tables are versioned, version the new table
+  IF EXISTS (SELECT p.oid FROM pg_catalog.pg_proc p,
+                               pg_catalog.pg_namespace n
+                          WHERE p.proname = 'ver_is_table_versioned'
+                          AND n.oid = p.pronamespace
+                          AND n.nspname = 'table_version')
+  THEN
+    IF table_version.ver_is_table_versioned('bde', 'crs_work')
```

Comment:
* @strk , why do you test if `crs_work` is versioned ? Just testing some random table if it is versioned ? Is it going to work if tables are installed without versions first and then after first L0 upload versions are added ?

Reply:
Yes, this is just a random table. I'm following what was already done in linz-lds-bde-schema here: https://github.com/linz/linz-lds-bde-schema/blob/1.10.1/sql/90-patches.sql#L111

If versioning is enabled later, it should include versioning of the new table (versioning is enabled for all tables under the `bde` schema: https://github.com/linz/linz-bde-schema/blob/1.10.1/sql/versioning/01-version_tables.sql#L46)